### PR TITLE
Update githubmodule update

### PR DESCRIPTION
**Pull request by pglodowski**
I'd like to review this with you when you're free

### CODE REVIEW

**Review by johnsarie27 — COMMENTED** [review]
Can you update the Install-GitHubModule to install without a version number if the module doesn't currently exist? This is the way it normally works. Let me know if you'd like to discuss and we can take a look.

File: Public/Install-GitHubModule.ps1 [hunk 1]
```diff
@@ -77,27 +77,34 @@ function Install-GitHubModule {
...
+                # UNBLOCK MODULE
+                if ($IsWindows -or $IsMacOS) {
+                    Write-Verbose -Message 'Unblocking files...'
+                    Get-ChildItem -Path $modulePath -Recurse | Unblock-File
+                }
             }
 
             # VALIDATE INSTALL
-            if (Test-Path -Path $modulePath -PathType Container) {
+            if (Get-Module -Name $Repository) {
```
Comment (johnsarie27) [inline Public/Install-GitHubModule.ps1:102]: This validates that the module is there, but not the version.  Do you want to add a check for the version too?

Comment (pglodowski) [reply Public/Install-GitHubModule.ps1]: This is fixed. All validates now check for the specific version number, in addition to the module itself.

**Review by johnsarie27 — APPROVED** [review]
reviewed with Phillip. all looks good

**Pull request merged**